matter/si91x/siwx917/BRD4338A/autogen/sl_event_handler.c

```diff
@@ -1,21 +1,25 @@
 #include "sl_event_handler.h"
 

```

Comment:
lots of styling changes. The restyler isn't inline with what slc generates, maybe we should remove restyler bot from this repo.
@mkardous-silabs thoughts?

Reply:
I'll open a PR to exclude the 917 generated files from the restyler for now.

Reply:
@chirag-silabs You will need to rebase and re-run the generation or delete restyler commits

Reply:
Will do!